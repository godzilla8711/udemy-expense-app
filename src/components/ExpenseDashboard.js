import React from 'react';
import ReactDOM from 'react-dom';

class ExpenseDashboard extends React.Component {
  render() {
    return (
      <div>
        This is the Expense Dashboard Component.
      </div>
    );
  }
}

export default ExpenseDashboard;
