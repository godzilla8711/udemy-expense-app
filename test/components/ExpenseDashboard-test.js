import React from 'react';
import { shallow } from 'enzyme';

import ExpenseDashboard from '../../src/components/ExpenseDashboard';

describe('render', () => {
  it('renders component', () => {
    const wrapper = shallow(<ExpenseDashboard />);
    expect(wrapper).toMatchSnapshot();
  });
});
