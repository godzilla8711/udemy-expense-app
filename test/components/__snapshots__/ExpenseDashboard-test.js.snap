// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render renders component 1`] = `
<div>
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseList) />
</div>
`;
